net/rp-pppoe/Makefile

```diff
@@ -29,7 +29,7 @@ define Package/rp-pppoe/Default
   SECTION:=net
   CATEGORY:=Network
   TITLE:=PPPoE (PPP over Ethernet)
-  URL:=https://dianne.skoll.ca/projects/rp-pppoe/
+  URL:=https://github.com/dfskoll/rp-pppoe
```

Comment:
Why such change? This is not described in the commit message.

Reply:
> Why such change? This is not described in the commit message.

The public Git repository mentioned on the author's homepage
https://dianne.skoll.ca/projects/rp-pppoe/

New code Github repositories mentioned on the RP-PPPoE archive list on the author's homepage
https://dianne.skoll.ca/pipermail/rp-pppoe/2021q3/thread.html
https://dianne.skoll.ca/pipermail/rp-pppoe/2021q4/thread.html